### DISCUSSION TITLE
Is there some way we can ask folks to include key information in bug reports?

### DISCUSSION THREAD

**Discussion by sbc100** [emscripten-core/emscripten#15732] [Ideas]
I've noticed that for the vast majority of bugs that get filed I end up asking:  

- What version of emscripten are you using?  (did it come from emsdk or some other source?)
- What the entire link command you are using?

Can we find a way to encourage all bug filers to include this information?  Perhaps its as simple as a bug template?

**Comment by kripken** [comment]
Bug template sounds like a good idea!

**Comment by sbc100** [comment]
I created basic template here: https://github.com/emscripten-core/emscripten/commit/85fdb448dd60c419a1ddc597c7ca8adc921383eb
